lib/harvester/base.rb

```diff
@@ -7,12 +7,18 @@ class Base
     def harvest_all(options = {})
       total = authors_query.count
       count = 0
-      logger.info "Started a complete harvest for #{total} authors at #{Time.zone.now}"
+      n = 0
+      start_time = Time.zone.now
+      logger.info "***** Started a complete harvest for #{total} authors at #{start_time}"
```

Comment:
This logging logic doesn't belong in the base class, IMO.  Also, you can use `each_with_index` instead of declaring and incrementing a variable yourself.

Reply:
Good point, switched to each_with_index.

Why don't you like this in the base class?  harvest_all is only implemented here, so i don't know where else to put global logging for a full scheduled harvest (which is very useful for monitoring)